### PR TITLE
feat(colors): update red color ramp

### PR DESCRIPTION
**Pull request by juliodialpad**
## Description
Updated red color ramp

## Pull Request Checklist

 - [x] Ask the contributors if a similar effort is already in process or has been solved.
 - [x] Review the [contribution guidelines](https://github.com/dialpad/dialtone/blob/staging/.github/CONTRIBUTING.md).
 - [ ] Use `staging` as your pull request's base branch. (All PRs using `production` as its base branch will be declined).
 - [x] Ensure all `gulp` scripts successfully compile.
 - [x] Update, remove, or extend all affected documentation.
 - [x] Ensure no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).

### CODE REVIEW

**Review by francisrupert — APPROVED** [review]
As with Green, I've added items to [DT-648](https://dialpad.atlassian.net/browse/DT-648) (especially for Danger Buttons 👀). But good and approved!

File: lib/build/svg/system/recording-off.svg [hunk 1]
```diff
@@ -1,4 +1,4 @@
 <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path d="M8 8H16V16H8V8Z" fill="#D81616"/>
-<path fill-rule="evenodd" clip-rule="evenodd" d="M12 4C7.58172 4 4 7.58172 4 12C4 16.4183 7.58172 20 12 20C16.4183 20 20 16.4183 20 12C20 7.58172 16.4183 4 12 4ZM2 12C2 6.47715 6.47715 2 12 2C17.5228 2 22 6.47715 22 12C22 17.5228 17.5228 22 12 22C6.47715 22 2 17.5228 2 12Z" fill="#D81616"/>
+<path d="M8 8H16V16H8V8Z" fill="#FFDCDC"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M12 4C7.58172 4 4 7.58172 4 12C4 16.4183 7.58172 20 12 20C16.4183 20 20 16.4183 20 12C20 7.58172 16.4183 4 12 4ZM2 12C2 6.47715 6.47715 2 12 2C17.5228 2 22 6.47715 22 12C22 17.5228 17.5228 22 12 22C6.47715 22 2 17.5228 2 12Z" fill="#FFDCDC"/>
 </svg>
```
Comment (francisrupert) [inline lib/build/svg/system/recording-off.svg]: @juliodialpad,

I noticed in the Magenta updates, the red in some icons seemed to be accounted for, but in this Red update they're obviously off.

image

Comment (francisrupert) [reply lib/build/svg/system/recording-off.svg]: Additionally, do we need to update the Gulp `buildSystemSVGs()`?

Comment (juliodialpad) [reply lib/build/svg/system/recording-off.svg]: Good catch, I missed to properly replace values in migration guide, now should be fixed.

Comment (juliodialpad) [reply lib/build/svg/system/recording-off.svg]: > Additionally, do we need to update the Gulp `buildSystemSVGs()`?

There's no need to, the values it removes are still the same so it continues to work just fine.

**Review by francisrupert — APPROVED** [review]

**Pull request merged**